### PR TITLE
Updated installation section with HACS

### PR DESCRIPTION
**Pull request by wsw70**
`auto-entities`  can be installed via HACS, I added a separate `Installation` section, split into HACS and manual (the existing one)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,23 @@ auto-entities
 
 Automatically populate lovelace cards with entities matching certain criteria.
 
-For installation instructions [see this guide](https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins).
+## Installation
+
+### HACS
+
+Search for for `auto-entities` in Plugins. After installing restart Home Assistant.
```
Comment (ludeeus) [inline README.md]: "After installing restart Home Assistant." That is not needed.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,23 @@ auto-entities
...
+
+In order to use it in the UI, add the following in the `resources` section
+
+```yaml
+resources:
+- url: /community_plugin/lovelace-auto-entities/auto-entities.js
```
Comment (ludeeus) [inline README.md]: `community_plugin` is wrong, that should be `hacsfiles`

File: README.md [hunk 3]
```diff
@@ -5,7 +5,23 @@ auto-entities
...
+  type: js
```
Comment (ludeeus) [inline README.md]: The type should be `module`

File: README.md (hunk 1 above)
Comment (wsw70) [reply README.md]: Yes it is, at least in my case (docker based HA installation).

File: README.md (hunk 2 above)
Comment (wsw70) [reply README.md]: At least for the docker HA installation, `community_plugin` is the right path
```
resources:
- type: js
  url: /community_plugin/rgb-light-card/card.js
- url: /community_plugin/lovelace-auto-entities/auto-entities.js
  type: js
```

File: README.md (hunk 3 above)
Comment (wsw70) [reply README.md]: It is `js` in my case and it works. I do not know HA enough to decide whether this is important or not.

**Comment by wsw70** [comment]
> The things added in this PR is not correct

It works perfectly in my HA installation which runs in a docker file. I do not know if this makes a difference (it should not) but calling is **incorrect** may be limited to your installation only, or to some installations.

I would not have gone through a PR to provide random, unverified information.

> and the information added is already at the bottom of the information inside HACS, that should be enough.

Or not. It depends whether you are looking for installation does useful to everyone or only to some.  This is a matter of style - I prefer to be inclusive just in case.

**Comment by ludeeus** [comment]
@wsw70 it works (for now), `js` as a type will be removed from HA, `community_plugin` will be removed from HACS (if you are running HA 0.107+ HACS will print a nasty warning if you still use the old URL).
As of now with the latest versions of HACS and HA it should use `hacsfiles` and `module`

What I'm saying is that the information is there already:
image
If you install with HACS, you are on the page that displays that information.

**Comment by wsw70** [comment]
@ludeeus 

I use 0.107.6 and took all the points you mentioned:

- I changed `js` with `module` → it works. I did not see the warning in HACS though
- I used `hacsfile` instead of `community_plugin` → it also works without problems.

OK in that case my PR is more or less moot and I will close it.

One comment though: maybe it is obvious for those who are used to HA and HACS, but the current installation part is I believe somehow misleading:

image

The installation link is OK (and it mentions HACS) but what is below ("install `auto-entities`as ...") is not clear. It is a specific case which should be explained in the installation page. BTW that page uses `js` and not `module`.

What I am trying to say is that installation directions should cover all reasonable cases, including people who are literate in computers in general but may not be experts in that particular area (such as myself)

**Pull request closed without merging**